tests/integration/conftest.py

```diff
@@ -67,17 +68,17 @@ class LLVMException(Exception):
     pass
 
 
-@pytest.fixture
-def run_int_fn():
-    def f(hugr: Package, expected: int, fn_name: str = "main"):
+def _run_fn(run_fn_name: str):
+    def f(hugr: Package, expected: Any, fn_name: str = "main"):
         try:
             import execute_llvm
 
-            if not hasattr(execute_llvm, "run_int_function"):
+            fn = getattr(execute_llvm, run_fn_name)
+            if not fn:
                 pytest.skip("Skipping llvm execution")
```

Comment:
```suggestion
                pytest.skip(f"Skipping llvm execution, {run_fn_name} not found")
```

Note we've already checked for `execute_llvm` not being imported, so we don't really expect this to fail (do we?)